src/app/notifications/types.ts

```diff
@@ -27,6 +27,11 @@ export interface ToastNotification {
   shouldAutoDismiss: boolean;
 }
 
+export interface ToastMeta {
+  destination: ToastNotification['destination'];
+  shouldAutoDismiss?: boolean;
+}
+
```

Comment:
`ToastNotification` extends `ToastMeta` will be handled in a different PR?

Reply:
yes